gentest/fixtures/YGDisplayTest.html

```diff
@@ -0,0 +1,20 @@
+<div id="display_none" style="width: 100px; height: 100px; flex-direction: row;">
```

Comment:
All these tests are a bit more complicated than they need to be. For example this first test could be done with just 2 children and have the same effect.